core/jvm/src/main/scala/fs2/interop/flow/syntax.scala

```diff
@@ -84,4 +84,17 @@ object syntax {
     def subscribe(subscriber: Subscriber[A])(implicit F: Async[F]): F[Unit] =
       flow.subscribeStream(stream, subscriber)
   }
+
+  private[flow] final class FromPublisherPartiallyApplied[F[_]](private val dummy: Boolean)
```

Comment:
Unfortunately this has to be public, otherwise MiMa will not support us here. I've been bitten by this more than once.
```suggestion
  final class FromPublisherPartiallyApplied[F[_]](private val dummy: Boolean)
```

Reply:
Also annoying if you can't see the scaladoc for this class while it's actually public API.